web/packages/teleterm/src/ui/ModalsHost/modals/HardwareKeys/Touch.tsx

```diff
@@ -55,7 +52,15 @@ export function Touch(props: {
           `}
         >
           <Image mb={4} width="200px" src={svgHardwareKey} />
-          <P2 bold>Touch your YubiKey</P2>
+          <P2 bold>
+            Touch your YubiKey to continue
+            {props.req.command && ' with command:'}
+            <br />
+            {
+              props.req.command &&
+                props.req.command /**TODO(Joerger): style text */
+            }
```

Comment:
```suggestion
            {props.req.command}
```

if this component is only showing some optional text, you dont have to do a check for it first, it'll just not render. even if the block is styled after your TODO, it wont render any block if the components arent styled (unless youre adding set widths and padding or whatever).

you would only need to check for the existence if it was together with that `with command` part above but since its on its own, just toss it out.

however, it makes more sense to combine those anyway and you can get rid of the `<br />` with someting like

```ts
      <P2 bold>
        Touch your YubiKey to continue
        {props.req.command && (
          <>
            {' with command'}
            <pre>{props.req.command}</pre>
          </>
        )}
      </P2>

```